hazelcast/src/main/resources/hazelcast-config-3.10.xsd

```diff
@@ -1821,6 +1821,25 @@
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
+            <xs:element name="merge-policy" type="merge-policy" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation>
+                        While recovering from split-brain (network partitioning), ScheduledExecutor tasks in the small cluster
+                        merge into the bigger cluster based on the policy set here.
+                        When an entry merges into the cluster, an entry with the same key might already exist in the cluster.
+                        The values of these entries might be different for that same key. Which value should be set for the
+                        key? The conflict is resolved by the policy set here.
+                        <p>
+                            <br/>There are built-in merge policies, such as:
+                            <br/>`com.hazelcast.cache.merge.PassThroughCacheMergePolicy` or `PASS_THROUGH`:
```

Comment:
I removed this documentation completely in my PRs, since it's already done in the `merge-policy` type itself. So we have it in a single place now and this should be redundant. Can be simplified to:
```xml
<xs:element name="merge-policy" type="merge-policy" minOccurs="0" maxOccurs="1"/>
````